pyproject.toml

```diff
@@ -29,11 +29,13 @@ classifiers = [
     "Typing :: Typed",
 ]
 dependencies = [
+    "click>=7.0",
```

Comment:
Can we avoid the dependency :confused: ?

Reply:
It is already a dependency so it doesn't matter. "mkdocstrings -> mkdocs -> click" will always be there

Reply:
Hmmm, true, thanks for the reminder.